src/frontend/next/src/components/header/Header.tsx

```diff
@@ -0,0 +1,215 @@
+import MenuIcon from '@material-ui/icons/Menu';
+import Link from 'next/link';
+import React, { useState } from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+// let MenuIcon: any = MenuIcon;
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      margin: 0,
+      padding: 0,
+      height: '4.5em',
+      width: '100%',
+      backgroundColor: '#333e64',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      position: 'fixed',
+      top: 0,
+    },
+    header_logo: {
+      flex: 1,
+    },
+    logo: {
+      height: '3.5em',
+      width: 'auto',
+      margin: '0.5rem 0.5rem',
+    },
+    header_navigation: {
+      padding: '5rem 0',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    search: {
+      flex: 1,
+      height: '1.8em',
+      width: 'auto',
+      filter: 'invert(85%)',
+    },
+    menu: {
+      flex: 3,
+      marginRight: '0.5rem',
+      display: 'flex',
+      justifyContent: 'space-between',
+      [theme.breakpoints.up(768)]: {
+        display: 'none',
+      },
+    },
+    menu_nav_ul_li: {
+      color: '#e5e5e5',
+      fontWeight: 500,
+      display: 'inline',
+      padding: '20px 15px',
+    },
+    menu_icon: {
+      display: 'none',
+      [theme.breakpoints.up(768)]: {
+        height: '1.5em',
+        width: 'auto',
+        filter: 'invert(85%)',
+        display: 'block',
+        margin: '0 1em',
+        flex: 1,
+      },
+    },
+    side: {
+      [theme.breakpoints.up(768)]: {
+        position: 'fixed',
+        top: 0,
+        left: 0,
+        bottom: 0,
+        right: 0,
+        content: ' ',
+        background: 'rgba(0, 0, 0, 0.5)',
+      },
+    },
+    sideNav: {
+      display: 'none',
+      [theme.breakpoints.up(768)]: {
+        margin: 0,
```

Comment:
I this should be `theme.breakpoints.down(768)` means that if the screen is anything less than 768 then the sideNav appears. What you are doing here is the right approach but with `up` if will match any screen size more than 768 which is not mobile.